packages/frontend/src/assets/css/globalStyles.css

```diff
@@ -33,6 +33,11 @@ div.v-application .theme--light.v-btn.v-btn--disabled .v-icon {
   color: var(--vscode-button-foreground, #ffffff) !important;
 }
 
+div.v-application i.theme--light.v-icon {
+  font-size: 20px;
+  color: var(--vscode-foreground, #cccccc);
+}
```

Comment:
consider to get approved from UX team (Cohen-Zur, Nirit)
does it applicable for all color theme ?

Reply:
Yes, `vscode-foreground` is the correct theme color.

Reply:
Replaced with `--vscode-notificationsInfoIcon-foreground`